src/LavaPlayerExtensions.cs

```diff
@@ -76,13 +76,18 @@ public static class LavaPlayerExtensions {
     /// </summary>
     /// <exception cref="NotImplementedException"></exception>
     public static async ValueTask StopAsync<TLavaPlayer, TLavaTrack>(this LavaPlayer<TLavaTrack> lavaPlayer,
-                                                                     LavaNode<TLavaPlayer, TLavaTrack> lavaNode)
+                                                                     LavaNode<TLavaPlayer, TLavaTrack> lavaNode,
+                                                                     TLavaTrack lavaTrack,
+                                                                     bool noReplace = false,
+                                                                     bool shouldPause = true)
         where TLavaTrack : LavaTrack
         where TLavaPlayer : LavaPlayer<TLavaTrack> {
         await lavaNode.UpdatePlayerAsync(
             lavaPlayer.GuildId,
+            noReplace,
             updatePayload: new UpdatePlayerPayload(
-                EncodedTrack: null));
+                EncodedTrack: lavaTrack.Hash,
```

Comment:
API: https://lavalink.dev/api/rest#update-player

```
* encodedTrack and identifier are mutually exclusive and deprecated. Use track instead.
```

Why are we adding a `shouldPause` arg? Shouldn't `noReplace` take care of this?